SimpleWAWS/Global.asax.cs

```diff
@@ -181,6 +195,19 @@ protected void Application_AuthenticateRequest(Object sender, EventArgs e)
                     SecurityManager.HandleAnonymousUser(context);
                 }
             }
+            else //coming in from auth provider
```

Comment:
what does `//coming in from auth provider` mean? this is an else for `if (!SecurityManager.TryAuthenticateSessionCookie(context))` which means the authentication was successful.